command.go

```diff
@@ -31,16 +121,81 @@ func NewExecCommand(exec string, args []string, showResult bool, description str
 	}
 }
 
+// Run runs the exec command
 func (c ExecCommand) Run() (string, error) {
+	config := readConfigOrDefault()
+
+	if config.DryRun {
+		return fmt.Sprintf("Dry Run: would have ran `%s` with args: %s", c.exec, c.args), nil
+	}
+
+	if config.Paused {
+		return fmt.Sprintf("Paused. But would have ran `%s` with args: %s", c.exec, c.args), nil
```

Comment:
Have fun with responses if you want.... Like more human like :)

For paused:
"I'm paused so I can't do that, but I would have run ...

For dry run:
"Doing that would run ...

or whatever you think.. up to you